src/comlink.ts

```diff
@@ -100,24 +100,23 @@ export const transferHandlers = new Map<string, TransferHandler>([
   [
     "throw",
     {
-      canHandle: obj => throwSet.has(obj),
-      serialize(obj) {
-        const isError = obj instanceof Error;
-        let serialized = obj;
+      canHandle: obj => typeof obj === "object" && throwMarker in obj,
```

Comment:
This currently would throw on
```ts
throw null;
```
I think. I fixed it in https://github.com/GoogleChromeLabs/comlink/pull/452